template_memo.html

```diff
@@ -0,0 +1,337 @@
+			}
+
+			.preheader {
+				display: none !important;
+				visibility: hidden;
+				opacity: 0;
+				color: transparent;
+				height: 0;
+				width: 0;
+			}
+
+			ul {
+				margin-top: 0px;
+				margin-bottom: 0px;
+			}
+
+			li {
+				margin-bottom: 15px;
+				font-family: Arial, "Helvetica Neue", sans-serif;
+				font-size: 16px;
+				line-height: 16px;
+				list-style-type: disc;
+			}
+
+			td.footer-text a {
+				font-weight: 400 !important;
+			}
+
+			table.emailwrapto100pc {
+				width: 600px;
+			}
+
+			td.button {
+				padding: 12px 18px !important;
+			}
+
+			@media screen and (max-width: 480px) {
+				table.emailwrapto100pc {
+					width: 100% !important;
+					white-space: inherit !important;
+					line-height: inherit !important;
+				}
+				/* td.logo-spacing {
+					padding-left: 20px !important;
+				} */
+				td.emailbodytext {
+					font-size: 16px !important;
+					line-height: 23px !important;
+					/* padding-right: 20px !important;
+					padding-left: 20px !important; */
+				}
+				.image-resize {
+					width: 100% !important;
+				}
+				table.footer-left,
+				table.footer-right {
+					width: 100% !important;
+					/* margin-top: 10px !important; */
+				}
+				/*table.footer-right {
+					float: left !important;
+					 margin-top: 25px !important; 
+				}*/
+				/* td.footer-right {
+					padding: 0px !important;
+				} */
+				td.footer-text {
+					padding: 30px 20px 20px 20px !important;
+				}
+				td.button-wrapper {
+					display: inline-block;
+				}
+				.outer-sidebox,
+				.inner-sidebox {
+					float: none !important;
+					width: 100% !important;
+				}
+				.sidebox-padding {
+					padding-left: 0px !important;
+				}
+			}
+			/* DARK MODE FOR IOS AND APPLE MAIL */
+			@media (prefers-color-scheme: dark) {
+				body {
+					/* background: #1c1c1e; */
+				}
+				a,
+				a:visited,
+				a:hover,
+				a:active {
+					color: #2d86e0;
+				}
+				a.button-text,
+				a.button-text:visited,
+				a.button-text:hover,
+				a.button-text:active {
+					color: #cecece;
+				}
+				td.emailbodytext p,
+				td.emailbodytext li,
+				td.emailbodytext h1,
+				td.emailbodytext h2,
+				td.emailbodytext h3,
+				td.tagline p,
+				td.dm-box h2 {
+					color: #cecece;
+				}
+				td.dm-box {
+					background: #2a2a2b !important;
+					color: #cecece;
+				}
+				td.emailbodytext {
+					/* border-bottom: 1px solid #2a2a2b !important; */
+				}
+				table.inner-sidebox,
+				td.dm-box {
+					border: 0 !important;
+				}
+			}
+		</style>
+	</head>
+	<body>
+		{include "dataLoader"}
+
+		<!-- <span class="preheader" style="display: none !important; visibility: hidden; opacity: 0; color: transparent; height: 0; width: 0">ENTER PREHEADER TEXT MESSAGE.</span>{include "pre-header-spacing"} --> {beacon}
+
+		<table role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" class="emailwrapto100pc" width="600px">
+			<tbody>
+				<!-- <tr>
+					<td align="center" style="padding: 20px 0">
```

Comment:
drop include to its own line. This way it makes it easier for local testing you can comment out with a keyboard shortcut